src/main/java/net/dv8tion/jda/core/hooks/AnnotatedEventManager.java

```diff
@@ -58,25 +59,20 @@ public void handle(Event event)
         Class<? extends Event> eventClass = event.getClass();
         do
         {
-            Map<Object, Method> listeners = methods.get(eventClass);
+            Map<Object, List<Method>> listeners = methods.get(eventClass);
             if (listeners != null)
             {
-                listeners.entrySet().forEach(e -> {
-                    try
-                    {
-                        e.getValue().setAccessible(true);
-                        e.getValue().invoke(e.getKey(), event);
-                    }
-                    catch (IllegalAccessException | InvocationTargetException e1)
-                    {
+                listeners.entrySet().forEach(e -> e.getValue().forEach(method -> {
+                    try {
```

Comment:
New line `{`, `}`, `if` and `else` everywhere! (also for lambdas even though it wasn't in the code before)